main/src/main/scala/sbt/internal/SysProp.scala

```diff
@@ -155,6 +155,8 @@ object SysProp {
   def useLog4J: Boolean = getOrFalse("sbt.log.uselog4j")
   def turbo: Boolean = getOrFalse("sbt.turbo")
   def pipelining: Boolean = getOrFalse("sbt.pipelining")
+  // opt-in or out of Zinc's consistent Analysis format.
+  def analysis2024: Boolean = getOrFalse("sbt.analysis2024")
```

Comment:
Since sbt 2.x plans to use consistent analysis format by default, we made not need it?

Reply:
Addressed in 12db63951b9a34c196622dbf7b8a7fdfe679435a